test/oComponentBoilerplate.test.js

```diff
@@ -0,0 +1,9 @@
+import expect from 'expect.js';
+
+const oComponentBoilerplate = require('./../src/js/componentBoilerplate');
+
+describe("oComponentBoilerplate", () => {
+	it('is defined', () => {
+		expect(oComponentBoilerplate).to.be.a('function');
+	});
```

Comment:
if you add an init function, probably worth adding a similar test for it https://github.com/Financial-Times/o-header/blob/master/test/header.test.js#L11